pkg/backend/snapshot.go

```diff
@@ -83,23 +85,20 @@ func (sm *SnapshotManager) Close() error {
 //
```

Comment:
Can you update the docs here to mention the new `bool` return type, and that it won't persist the snapshot if the callback returns false?

Reply:
Yep.